src/components/modules/MediaForm.svelte

```diff
@@ -8,10 +8,10 @@
 </script>
 
 <div class="media-form-container" class:expanded>
-    <div class="module_topbar" on:click={() => {expanded = !!!expanded}}>
-      <div class="module_title text_level1"><div class="arrow">«</div>Options</div>
+    <div class="module_topbar">
+      <div class="module_title text_level1" on:click={() => {expanded = !!!expanded}}><div class="arrow">«</div>Options</div>
     </div>
-    <div class="media-form-conteant" class:expanded>
+    <div class="media-form-content" class:expanded>
```

Comment:
typo, needs it's own commit and be applied as a fixup to the former one.

Reply:
Resolved in new PR